clippy_lints/src/await_holding_invalid.rs

```diff
@@ -73,11 +77,11 @@ declare_clippy_lint! {
 
 declare_clippy_lint! {
     /// ### What it does
-    /// Checks for calls to await while holding a `RefCell` `Ref` or `RefMut`.
+    /// Checks for calls to `await` while holding a `RefCell`, `Ref`, or `RefMut`.
```

Comment:
I've tried to standardise the usage here so that whenever we refer to the keyword `await`, it's in code style, whereas when we're talking about things it applies to (such as "await points") it's not.

This matches the existing style with regard to `async`: when used as a keyword, it's in code style, but when used as an adjective ("async function") it's not.